devito/core/gpu_openmp.py

```diff
@@ -14,6 +14,16 @@
                                ParallelTree, optimize_halospots, mpiize, hoist_prodders,
                                iet_pass)
 from devito.tools import as_tuple, filter_sorted, timed_pass
```

Comment:
Some general guidelines:
some of this is written here: https://github.com/devitocodes/devito/blob/master/CONTRIBUTING.md#coding-guidelines